src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,43 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String engine;
+    private int maxSpeedKmH;
+    private boolean isEcoFriendly;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String engine, int maxSpeedKmH, boolean isEcoFriendly) {
+        this.engine = engine;
+        this.maxSpeedKmH = maxSpeedKmH;
+        this.isEcoFriendly = isEcoFriendly;
+    }
+
+    public String getEngine() {
+        return engine;
+    }
+
+    public void setEngine(String engine) {
+        this.engine = engine;
+    }
+
+    public int getMaxSpeedKmH() {
+        return maxSpeedKmH;
+    }
+
+    public void setMaxSpeedKmH(int maxSpeedKmH) {
+        this.maxSpeedKmH = maxSpeedKmH;
+    }
+
+    public boolean isEcoFriendly() {
+        return isEcoFriendly;
+    }
+
+    public void setEcoFriendly(boolean ecoFriendly) {
+        isEcoFriendly = ecoFriendly;
+    }
```

Comment:
![image](https://github.com/mate-academy/jv-pecs/assets/113807446/30c6911d-691c-4094-a95c-da717d273396)
I thought here they wanted me to add access modifiers + access methods even in case I don't use them for the task.